### PR TITLE
[#854][FOLLOWUP] feat(tez): Add RssTezFetcherTask to fetch data from worker for OrderedInput

### PR DESCRIPTION
**Pull request by lifeSo**
### What changes were proposed in this pull request?

Add RssTezFetcherTask to fetch data from worker for OrderedInput

### Why are the changes needed?

Fix: https://github.com/apache/incubator-uniffle/issues/854

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

unit test

### CODE REVIEW

**Comment by lifeSo** [comment]
we will push unit test case in this PR later.

**Comment by lifeSo** [comment]
"Too many files with unapproved license"

This is un-related to my commit ?

**Comment by jerqi** [comment]
It's caused by your pr. Your file don't include Apache license header.

**Comment by lifeSo** [comment]
> @lifeSo It's caused by your pr. Your file don't include Apache license header.

ok, I understand, I re-commit file, please check again.

**Comment by jerqi** [comment]
> > @lifeSo It's caused by your pr. Your file don't include Apache license header.
> 
> ok, I understand, I re-commit file, please check again.

Could you add ut for this class?

**Comment by lifeSo** [comment]
> > > @lifeSo It's caused by your pr. Your file don't include Apache license header.
> > 
> > 
> > ok, I understand, I re-commit file, please check again.
> 
> @lifeSo Could you add ut for this class?

Tez code  耦合性 is too big and test case is not easy to write...

**Comment by jerqi** [comment]
> > > > @lifeSo It's caused by your pr. Your file don't include Apache license header.
> > > 
> > > 
> > > ok, I understand, I re-commit file, please check again.
> > 
> > 
> > @lifeSo Could you add ut for this class?
> 
> Tez code 耦合性 is too big and test case is not easy to write...

You can refer to MR test code.

File: client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssTezShuffleDataFetcher.java [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  private long startWait;
+  private int waitCount = 0;
+  private byte[] uncompressedData = null;
+  private final Codec rssCodec;
+  private Integer partitionId;
+  private final ExceptionReporter exceptionReporter;
+
+  private final AtomicInteger issuedCnt = new AtomicInteger(0);
+
+  public RssTezShuffleDataFetcher(InputAttemptIdentifier inputAttemptIdentifier,
```
Comment (jerqi) [inline client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssTezShuffleDataFetcher.java:75]: Could we use the code style to keep consistent with other places?
```
RssTezShuffleDataFetcher(
        InputAttemptIdentifier inputAttemptIdentifier,
       Integer partitionId
       ...
```

Comment (lifeSo) [reply client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssTezShuffleDataFetcher.java:75]: We used, but the tool not find the code wrong, may recommend some tools please?

Comment (jerqi) [reply client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssTezShuffleDataFetcher.java:75]: > We used, but the tool not find the code wrong, may recommend some tools please?

Let's modify this place first.

Comment (lifeSo) [reply client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssTezShuffleDataFetcher.java:75]: ok, I modified it and pushed

**Review by jerqi — APPROVED** [review]
LGTM, thanks @lifeSo , merged to master.

**Pull request merged**